stable_baselines3/common/buffers.py

```diff
@@ -311,6 +313,10 @@ def _get_samples(self, batch_inds: np.ndarray, env: Optional[VecNormalize] = Non
         )
         return ReplayBufferSamples(*tuple(map(self.to_torch, data)))
 
+    @staticmethod
+    def _get_action_data_type(action_space):
```

Comment:
missing type annotations and docstring

Reply:
Ou, sorry for missing this. 
1. I added typing. 
2. I added a comment using `#` not a full docstring to stay consistent with other private methods.